src/App.js

```diff
@@ -9,31 +9,38 @@ import TopicContainer from './topic/TopicContainer';
 class App extends Component {
     render() {
         return (
-            <Main className="App">
+            <React.Fragment>
                 <TopBar/>
                 <Header/>
-                <WorkshopForm/>
-                <TopicContainer/>
-            </Main>
+                <main>
+                    <WorkshopForm/>
+                    <TopicContainer/>
+                </main>
+            </React.Fragment>
         );
     }
 }
 
-const Main = styled.main`
-    text-align: center;
-    box-sizing: border-box;
-    * {
-        box-sizing: inherit;
-    }
-`;
 
 /* eslint no-unused-expressions: 0 */
 injectGlobal`
-  @import url('https://fonts.googleapis.com/css?family=Lato');
-  
-  body {
-    font-family: 'Lato', Sans-Serif;
-  }
+    @import url('https://fonts.googleapis.com/css?family=Lato');
+
+    html {
+        box-sizing: border-box;
+    }
+
+    *,
+    *:after,
+    *:before {
+        box-sizing: inherit;
+    }
+
+    body {
+        margin: 0;
```

Comment:
How about putting it to `reset.css`?

Reply:
Yeah, the style is already in the `reset.css`, so it's duplicated code.

```
html, body, ... {
    margin: 0;
    padding: 0;
    border: 0;
    font-size: 100%;
    font: inherit;
    vertical-align: baseline;
}
```

I will remove the duplicated styles only this will stay:

```
injectGlobal`
    @import url('https://fonts.googleapis.com/css?family=Lato');

    html {
        box-sizing: border-box;
    }

    *,
    *:after,
    *:before {
        box-sizing: inherit;
    }

    body {
        font-family: 'Lato', sans-serif;
    }
`;
```

Reply:
Done. [Remove duplicated style](https://github.com/warsawjs/topics-manager/pull/15/commits/5b85b1a9fbc720faa06c464a8dc8b99a5b29812c)